haskell/private/haskell_impl.bzl

```diff
@@ -401,10 +401,29 @@ def haskell_library_impl(ctx):
             set.singleton(c_p.interfaces_dir),
         )
 
+    version_macros = set.empty()
```

Comment:
Can this be factored with the code in `haskell_import.bzl`?

Reply:
I've factored it out into `haskell/private/version_macros.bzl`.